app/assets/javascripts/customFilters.js.coffee

```diff
@@ -10,6 +10,15 @@ angular.module('customFilters', [])
     suffix = if relevantDigits <= 3 then suffixes[relevantDigits] else suffixes[0]
     dtfilter + suffix
   ]
+.filter 'dateSuffixOrShort', ['$filter', ($filter) ->
```

Comment:
is there a better name we could have for this that describes what it's doing?

Reply:
might also be helpful to add some comments to explain the business rules around this, as per @callaghanc's suggestion recently

Reply:
I've added a comment and slightly improved the name. I dont have a better idea, but I'm open for suggestions.